pkg/config/configsections/Nodes.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package configsections
+
+// Nodes defines in the cluster. with name of the node and the type of this node master/worker,,,,,
+type Nodes struct {
+	Name string
+	Type []string
```

Comment:
I think we have only two types, master and worker
you can use this 
```
type Node struct {
	// Name is the name of node
	Name string `yaml:"name" json:"name"`
	// Worker indicates if the node is defined as worker in OCP cluster
	Worker bool `yaml:"worker" json:"worker"`
	// Master indicates if the node is defined as master in OCP cluster
	Master bool `yaml:"master" json:"master"`
}
```
Also, this Node not Nodes.

Reply:
Two booleans? Isn't it simpler to have just two node list/array?

Reply:
we had a discussion about this. 
a node can have multiple types (storage, worker..) 
a better solution would be to store all those values in Type slice. But we will have two functions isMaster() and isWorker()
having two maps  is not good idea since some tests may need to loop through all the nodes and having two loops would make things long

Reply:
Sure, I just don't like two bools as it created invalid combinations.